### DISCUSSION TITLE
Παρουσίαση τελικής προόδου

### DISCUSSION THREAD

**Discussion by epidrome** [upatras-hci/iv#74] [Announcements]
Για τις επόμενες συναντήσεις μετά τις γιορτές θα γίνουν οι παρουσιάσεις τελικής προόδου, δλδ τουλάχιστον μέχρι και το παραδοτέο 10.

Για τον σκοπό αυτό, απαντήστε παρακάτω ποια εβδομάδα θέλετε να παρουσιάσετε την πρόοδο σας, ώστε μετά να συμπληρωθεί ο πίνακας:

| Εβδομάδα | Όνομα |
| -- | -- |
| 19 Ίαν | [1](https://github.com/e-liouta) [2](https://github.com/akimo13) |
| 26 Ίαν (Αναπλήρωση) | [1](https://github.com/EvangeliaChatz) [2](https://github.com/MairySs) |
| 13 Φεβ. | [1](https://github.com/EvangeliaChatz) 2 |

Ο σκοπός της παρουσίασης είναι να δείξετε την αναφορά σας και να γίνουν σχόλια βελτίωσης. Σε κάθε συνάντηση μπορούμε να έχουμε μόνο 1-2 παρουσιάσεις έτσι ώστε να υπάρχει χρόνος για να δούμε τις τρέχουσες δραστηριότητες και την θεωρία.

**Comment by e-liouta** [comment]
Καλησπέρα! Εγώ θα παρουσιάσω 19 Ιανουαρίου! 💃
Στις άλλες ημερομηνίες δεν έχουμε μάθημα, είναι βέβαια και αυτό που θα αναπληρώσουμε που όπως είπατε θα κάνουμε ψηφοφορία. Αν κάποιος επιλέξει τις άλλες ημερομηνίες θα παρουσιάσει μόνο σε εσάς;

**Comment by akimo13** [comment]
Καλημέρα σας, από τη στιγμή που αφορά παρουσίαση προόδου προκειμένου να έχουμε κάποια σχόλια για το τελικό μας παραδοτέο που πρόκειται να αξιολογηθεί και εμένα με βολεύει στις 19 Ιανουαρίου.

**Comment by EvangeliaChatz** [comment]
Καλημέρα σας, θα παρουσιάσω κι εγώ στις 26 Ιανουαρίου.

**Comment by MairySs** [comment]
Καλησπέρα, 26 Ιανουαρίου θα παρουσιάσω και εγώ.
